tests/llm/utils/port_forward.py

```
@@ -271,7 +292,11 @@ def setup_all_port_forwards(test_cases: List[HolmesTestCase]) -> PortForwardMana
             manager.add_port_forward(config)
 
         # Start all port forwards
-        manager.start_all()
+        try:
+            manager.start_all()
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)